doc/source/conf.py

```diff
@@ -101,6 +110,153 @@
+    "colorMaps",
+    "ConsoleWidget",
+    "contextMenu",
+    "crosshair",
+    "customGraphicsItem",
+    "CustomGraphItem",
+    "customPlot",  # doesn't show the triangles along the x axis
+    "DataSlicing",
+    "DataTreeWidget",  # wrapped c/c++ object of type QComboBox has been deleted
+    # "DateAxisItem",  # can't use __file__
+    # "DateAxisItem_QtDesigner",  # can't use __file__
+    # "designerExample",  # can't use __file__
+    "DiffTreeWidget",
+    "dockarea",
+    # "Draw",  # necessarily interactive
+    "ErrorBarItem",
+    # "FillBetweenItem",  # animated
+    "FlowchartCustomNode",
+    "Flowchart",
+    "fractal",
+    "glow",
+
+    # some GL examples do seem to work, but others give GL errors
+    "GLBarGraphItem",
+    "GLGradientLegendItem",
+    "GLGraphItem",
+    "GLImageItem",
+    "GLIsosurface",
+    "GLLinePlotItem",
+    "GLMeshItem",
+    "GLPainterItem",
+    "GLScatterPlotItem",
+    "GLshaders",
+    "GLSurfacePlot",
+    "GLTextItem",
+    "GLViewWidget",
+    "GLVolumeItem",
+
+    "GradientEditor",
+    "GradientWidget",
+    "GraphicsLayout",
+    "GraphicsScene",
+    "GraphItem",
+    "hdf5.py",
+    "HistogramLUT",
+    "histogram",
+    "imageAnalysis",
+    "ImageItem",
+    "ImageView",
+    "infiniteline_performance",
+    "InfiniteLine",
+    "isocurve",
+    "JoystickButton",
+    "Legend",
+    "linkedViews",
+    "logAxis",
+    "LogPlotTest",
+    "MatrixDisplayExample",
+    "MouseSelection",
+    "MultiplePlotAxes",
+    "multiplePlotSpeedTest",
+    "MultiPlotSpeedTest",
+    "MultiPlotWidget",
+    # "multiprocess",  # swallowed exception, not sure what's wrong
+    "NonUniformImage",
+    "optics_demos",
+    # "PanningPlot",  # animated
+    # "parallelize",  # hangs
+    "parametertree",
+    "PColorMeshItem",
+    "PlotAutoRange",
+    "PlotSpeedTest",
+    "Plotting",
+    "PlotWidget",
+    # "ProgressDialog",  # hangs
+    "relativity_demo",
+    # "RemoteGraphicsView",  # doesn't show any plot contents
+    # "RemoteSpeedTest",  # remoteproxy NoResultError
+    "ROIExamples",
+    "ROItypes",
+    "ScaleBar",
+    "ScatterPlot",
+    "ScatterPlotSpeedTest",
+    "ScatterPlotWidget",
+    "scrollingPlots",
+    "SimplePlot",
+    "SpinBox",
+    "Symbols",
+    "TableWidget",
+    "text",
+    "TreeWidget",
+    "verlet_chain_demo",
+    "VideoSpeedTest",
+    "ViewBoxFeatures",
+    "ViewBox",
+    "ViewLimits",
+]
+
+DEFAULT_CONFIG_OPTIONS = pyqtgraph.CONFIG_OPTIONS.copy()
+
+
+def reset_pg_config(gallery_conf, fname):
+    """sphinx-gallery reset callback to reset to default pg config"""
+    pyqtgraph.setConfigOptions(**DEFAULT_CONFIG_OPTIONS)
+
+
+def argv_handler(gallery_conf, script_vars):
+    if script_vars["src_file"].endswith("hdf5.py"):
+        return ["test.hdf5", "1000000"]
+    else:
+        return []
+
+
+# sphinx gallery config
+sphinx_gallery_conf = {
+    "examples_dirs": os.path.join(
+        os.path.dirname(__file__), "..", "..", "pyqtgraph", "examples"
```

Comment:
probably should wrap with `os.path.abspath` so the console/log output doesn't have directories with `..` throughout.